lib/test/cdo/test_web_purify.rb

```diff
@@ -26,8 +29,13 @@ def test_find_potential_profanities
   end
 
   def test_find_potential_profanities_with_language
-    assert_nil WebPurify.find_potential_profanities('scheiße', ['en'])
+    assert_nil WebPurify.find_potential_profanities('scheiße', ['es'])
```

Comment:
Since the fixtures were updated, this test had to change because WebPurify considers `scheiße` profanity in English and German.